### PR TITLE
Update OWNERS file in client-go/tools/leaderelection adding jefftree

### PR DESCRIPTION
**Pull request by fedebongio**
Adding jefftree to the owners file for leaderelection

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by jpbetz** [comment]
This seems appropriate to me given @Jefftree's role in Coordinated Leader Election.

/assign @deads2k @sttts for thoughts.

File: staging/src/k8s.io/client-go/tools/leaderelection/OWNERS [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 approvers:
   - mikedanese
+  - jefftree
```
Comment (BenTheElder) [inline staging/src/k8s.io/client-go/tools/leaderelection/OWNERS:5]:
```suggestion
  - jefftree
```

**Comment by BenTheElder** [comment]
context: in api machinery triage yesterday we noticed that there is only @mikedanese (who may be ~inactive now?), if there are other potential owners we should also consider adding them.

File: staging/src/k8s.io/client-go/tools/leaderelection/OWNERS [hunk 2]
```diff
@@ -2,6 +2,7 @@
...
```
Comment (aojea) [inline staging/src/k8s.io/client-go/tools/leaderelection/OWNERS:5]: I think you should also add jefftree to reviewers

**Comment by thockin** [comment]
Ping!  This has been open a long time, @fedebongio there is one comment

**Review by jpbetz — APPROVED** [review]
/lgtm
/approve

**Comment by fedebongio** [comment]
Done, sorry for the delay!

**Pull request merged**